demodata.json

```diff
@@ -1,296 +1,282 @@
 [

```

Comment:
do you mind indenting it like it was before (as output by dumpdata) so that the diff is more readable and we don't have to manually indent each time we dumpdata?